data/transactions/logic/opcodes.go

```diff
@@ -598,11 +651,57 @@ var OpSpecs = []OpSpec{
 	{0xd1, "block", opBlock, proto("i:a"), randomnessVersion, field("f", &BlockFields)},
 }
 
+// OpNames lists all spec names in order they are introduced in the OpSpecs table
+var OpNames = make([]string, 0)
+
 type sortByOpcode []OpSpec
 
-func (a sortByOpcode) Len() int           { return len(a) }
-func (a sortByOpcode) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
-func (a sortByOpcode) Less(i, j int) bool { return a[i].Opcode < a[j].Opcode }
+func (a sortByOpcode) Len() int      { return len(a) }
+func (a sortByOpcode) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a sortByOpcode) Less(i, j int) bool {
+	first := GetFullCode(a[i])
+	second := GetFullCode(a[j])
+	for k := range first {
+		if len(second) == k {
+			return false
+		}
+		if first[k] != second[k] {
+			return first[k] < second[k]
+		}
+	}
+	return true
+}
+
+// Serialize is used to write multi-byte opcodes as strings
+func Serialize(bytes []byte) string {
+	if len(bytes) == 0 {
+		return ""
+	}
+	ret := ""
+	for _, b := range bytes {
+		ret += fmt.Sprintf("0x%02x ", b)
+	}
+	return strings.TrimSpace(ret)
+}
+
+// IsMultiLeaf identifies if a spec is a multi-op, but it only functions after the spec has been annotated in init()
+func IsMultiLeaf(spec OpSpec) bool {
+	return spec.MultiCode != nil
+}
```

Comment:
I need to understand why this and `isMultiCode` both exist.

Reply:
isMultiOp (now named isMultiRoot) identifies if a spec has children
IsMultiLeaf identifies if a spec has a multi-byte opcode

Reply:
I see.  How about `isMultiPrefix` or even just `isPrefix` for the first?  I actually think of the second one as `isMultiOp`.